elastic/kibana
# [Response Ops] Fixing ES index connector so that it can index into data streams as well as indices.

Resolves https://github.com/elastic/kibana/issues/131295

## Summary

* Adds `op_type: create` to index call when using ES Index connector. This allows the action to work correctly on data streams

## To verify:
* Create a data stream
* Create a connector and test indexing a document into the data stream. Indexing should be successful and not fail.
* Verify that action still works with indices that are not data streams.

### Checklist
